src/SFA.DAS.TrackProgress.Api/Program.cs

```diff
@@ -0,0 +1,37 @@
+using SFA.DAS.TrackProgress.Api.AppStart;
+using SFA.DAS.TrackProgress.Api.Configuration;
+
```

Comment:
I've not seen us write code without putting it into a namespace?

Reply:
This is the new .Net6 [Top Level Program](https://docs.microsoft.com/en-us/dotnet/core/tutorials/top-level-templates).  That's how the Visual Studio new project template creates the code

Reply:
Interesting, Sonar is highlighting it as a bug!